pyclustering/core/pyclustering_package.py

```diff
@@ -232,6 +244,13 @@ def __create_package_numpy_matrix(self, dataset_package, dataset):
         return pointer(dataset_package)
 
 
+    def __create_package_string(self, dataset_package, string_utf8):

```

Comment:
- [ ] Cosmetic. String could have any encoding. Rename variable to `string_value`.